examples/crd-client/cloudbuild.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  #
+  # Creates the initial make + docker build platform
+  #
+  - name: "ubuntu"
+    script: |
+      echo 'FROM gcr.io/cloud-builders/docker\nRUN apt-get install make\nENTRYPOINT [\"/usr/bin/make\"]' > Dockerfile.build
+  - name: gcr.io/cloud-builders/docker
+    id: build-make-docker
+    entrypoint: "docker"
+    args: ["build", "-f", "Dockerfile.build", "-t", "make-docker", "."]
+
+  # build and push crd-client image to GCR
+  - name: "make-docker"
+    id: push
+    dir: "."
+    env:
+      - REPOSITORY=${_REPOSITORY}
+    script: |
+      make -j 1 push
+options:
+  machineType: E2_HIGHCPU_32
```

Comment:
```suggestion
```

I think a standard machine will do - it'll be faster too, as the bigger machines take longer to initially spin up.

Reply:
(Same for the other examples too)

Reply:
Just wanted to let you know that I noticed a slight difference in build completion times while testing with the xonotic image. Specifically, when using machineType: E2_HIGHCPU_32, the [build](https://pantheon.corp.google.com/cloud-build/builds;region=global/34f63ded-c03c-4f9a-a33a-1f47b019e197?mods=logs_tg_prod&project=agones-mangalpalli) took approximately 4 minutes, whereas without this machine type, [build](https://pantheon.corp.google.com/cloud-build/builds;region=global/ba88c356-7e20-4b12-9e44-7e32c284b629?mods=logs_tg_prod&project=agones-mangalpalli) took around 7 minutes.

Reply:
Yeah that's fine - there's often a longer wait for 32's to spin up too, so there is that as well.